### PR TITLE
website/integrations: add Paperless-ng instructions

### PR DESCRIPTION
**Pull request by Neriderc**
I have written instructions for using Paperless with authentik, using the forward-auth method.

### CODE REVIEW

**Comment by Neriderc** [comment]
Any advice on how to work out what the issue is here?

File: website/integrations/services/paperless-ng/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The author of Paperless-ng recommends you do not expose Paperless outside your network, as it was not designed for that. Instead, they "recommend that if you do want to use it, run it locally on a server in your own home."
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+- `paperless.company` is the FQDN of the Paperless-ng install.
+
```
Comment (BeryJu) [inline website/integrations/services/paperless-ng/index.md:24]: Add missing authentik.company placeholder

File: website/integrations/services/paperless-ng/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+PAPERLESS_HTTP_REMOTE_USER_HEADER_NAME=HTTP_X_AUTHENTIK_USERNAME
+```
+Authentik automatically sets this header when we use a proxy outpost.
+
+Now restart your container:
+`docker-compose down && docker-compose up -d`
+
+## authentik
+
+### Provider
```
Comment (BeryJu) [inline website/integrations/services/paperless-ng/index.md:42]: Remove these subcategories to match the other integrations

File: website/integrations/services/paperless-ng/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Create a Proxy Provider. Give it a name (e.g. `Paperless Proxy`), then choose explicit or implicit consent (whether you want authentic to show a button to proceed to Paperless after login, or to just go there).
+
+Choose Forward Auth (single application), then add the External host: `https://paperless.company` 
+
+Click Create to finish creating the provider.
+  
+### Application
+  
+Now go to Applications->Applications and create a new application.
```
Comment (BeryJu) [inline website/integrations/services/paperless-ng/index.md:53]: Use cursive text to refer to the applications menu

File: website/integrations/services/paperless-ng/index.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Set the slug, let's use `paperless`.
+  
+Now select the provider we created earlier, e.g. `Paperless Proxy`.
+  
+Click Create to create the application.
+ 
+### Outpost
+ 
+Now go to Applications->Outposts and click the edit button for "authentik Embedded Outpost".
```
Comment (BeryJu) [inline website/integrations/services/paperless-ng/index.md:65]: Same as above

File: website/integrations/services/paperless-ng/index.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Under Applications, click Paperless to select it (and select any other services in the list you want to keep working - use ctrl+click to select multiple), then click Update at the bottom.
```
Comment (BeryJu) [inline website/integrations/services/paperless-ng/index.md:67]: remove the `and select any other services in the list you want to keep working` part

**Review by BeryJu — CHANGES_REQUESTED** [review]
Some minor stuff, basically just consistency issues, see https://goauthentik.io/developer-docs/docs/writing-documentation

File: website/integrations/services/paperless-ng/index.md (hunk 1 above)
Comment (Neriderc) [reply website/integrations/services/paperless-ng/index.md:24]: As it uses forward-auth, which is already covered in the documentation, there's no place where you need to put authentik.company. Should it still be listed as a placeholder if it is never referenced?

File: website/integrations/services/paperless-ng/index.md (hunk 3 above)
Comment (Neriderc) [reply website/integrations/services/paperless-ng/index.md:53]: Sorry, is cursive text the same as _itallics_?

File: website/integrations/services/paperless-ng/index.md (hunk 1 above)
Comment (BeryJu) [reply website/integrations/services/paperless-ng/index.md:24]: I just checked https://goauthentik.io/integrations/services/tautulli/ for references, and that doesn't use the placeholder either but it's still there for consistency sake. Should be fine without

**Comment by Neriderc** [comment]
@BeryJu I believe all the requested changes have been made.

**Pull request merged**